### DISCUSSION TITLE
Record level Permissions

### DISCUSSION THREAD

**Discussion by arladmin** [Budibase/budibase#1210] [Ideas]
Hi. Can we also include Record Level Permissions (ACLs ~ Access Control Lists)?

The 'ACL' column would take, user/automation-defined, single/multiple values, each value symbolising a particular set of granular permissions.
e.g. 
- Read/Write/Delete access to the Creator of the record +
- Read/Write Access to all user belonging to Admin role +
- Read Access to all users belonging to Public role
etc.

A very nice working implementation of this (referencing another platform here, Parse) is:
http://docs.parseplatform.org/js/guide/#object-level-access-control
(object = record)

This would also needed to be reflected accordingly in the frontend, of course.

**Comment by joebudi** [comment]
Hey @arladmin .

Thank you for opening this discussion. @mjashanks has been pushing record-level permissions for a while. They are on the roadmap and we hope to implement in the future.

Thanks for the link to Parse!

**Reply by mjashanks** [reply]
It is very highly requested! We will definitely look into it at some point.
